rust/feature-flags/src/team/team_models.rs

```diff
@@ -4,15 +4,18 @@ use serde::{Deserialize, Serialize};
 // TODO: Add integration tests across repos to ensure this doesn't happen.
 pub const TEAM_TOKEN_CACHE_PREFIX: &str = "posthog:1:team_token:";
 
+pub type TeamId = i32;
+pub type ProjectId = i64;
```

Comment:
Hmm, does this do what my previous comment about strongly typed IDs?

Reply:
Sorta, I think it's lucky that they're different types, so I get an error when using the wrong type of data here (useful for app code).  However, it's not as handy for tests, for the reasons I explained above.